var/spack/repos/builtin/packages/pasta/package.py

```diff
@@ -19,6 +19,7 @@ class Pasta(Package):
     depends_on("python@3:", when="@1.9.0:")
     depends_on("python@:2", when="@:1.8.3")
     depends_on("py-dendropy")
+    depends_on("py-setuptools")
     depends_on("java")
```

Comment:
Guessing this should look something like:
```suggestion
    depends_on("python@3:", when="@1.9:", type=("build", "run"))
    depends_on("python@:2", when="@:1.8", type=("build", "run"))
    depends_on("py-dendropy@4:", type=("build", "run"))
    depends_on("py-setuptools", type="build")
    depends_on("java", type=("build", "run"))
```

Reply:
I'll try, or at the very least make the same changes we had with busco.